setup.cfg

```diff
@@ -1,12 +1,2 @@
-[mypy]
-
-[mypy-numpy,pandas.*,pytest,setuptools,tableauhyperapi.*,pyarrow.*]
-ignore_missing_imports=True
-
 [flake8]
```

Comment:
Would also be OK to get rid of flake8; seems like ruff is the most popular tool now